tests/views/tp.py

```diff
@@ -53,15 +53,15 @@ def _test_browse_view(tp, request, response, kwargs):
     pootle_path = "%s%s" % (tp.pootle_path, resource_path)
 
     if not (kwargs["dir_path"] or kwargs.get("filename")):
-        ob = tp.directory
```

Comment:
In the `if` clause we have `obj = Store.objects.get(pootle_path=pootle_path)` so they're not always directories.  So I felt I couldn't use that.